tests/tests/optionsTests.robot

```diff
@@ -0,0 +1,47 @@
+*** Settings ***
+Documentation       Options Tests
+
+Library             SeleniumLibrary
+Library             String
+Resource            src/page_objects/keywords/common.robot
+Resource            src/page_objects/keywords/home_page.robot
+Resource            src/page_objects/keywords/ubuntu.robot
+Resource            src/test_files/test_files.resource
+Library             src/page_objects/libraries/browser_lib.py
+
+Suite Setup         Prepare Test Environment
+Test Setup          Prepare For Test
+Test Teardown       Clean After Test
+
+*** Variables ***
+${TEST_FILE}                ${file_4mb}
+${EXTRACTION_TIMEOUT}       60s
+
+*** Test Cases ***
+Enable Debug Output functionality test    
+    [Documentation]    Enable debug output functionality adds debug logs if selected
+    [Tags]    options
+
+    ${downloaded_file_path}    Set Variable    ${DOWNLOAD_PATH}${TEST_FILE}[archive_name].zip
```

Comment:
I think it would be safer here to use "set test variable" instead of "set variable" here

Reply:
it turned out that this variable is not used in the test so I deleted that line